sdk/monitor/monitor-query/src/models/publicLogsModels.ts

```diff
@@ -66,6 +66,7 @@ export interface ErrorInfo extends Error {
   additionalProperties?: Record<string, unknown>;
 }
 
+/** Batch Error class */
```

Comment:
It's already obvious that this is a `BatchError` class - is it possible to extend this (and the `AggregateBatchError` doc comments with what it represents / when I can expect to see this vs AggregateBatchError for example?

(btw, the same ask would apply in `AggregateBatchError` - what sort of documentation would be useful for someone hovering over the type?)

Reply:
that when these are returned? I can add them

Reply:
added